ui/ui_vfinal/intents/javascript_functions/submit_query.html

```diff
@@ -152,45 +152,11 @@
       }
     }
     
-    // Creating suggestions list
-    var messages = '';
-    for(var i = 0; i < suggestions.length; i++) {
-      messages += '<div class="suggestion-box">';
-      
-      // Adding suggestions text
-      messages += '<span class="suggestion-text">' + suggestions[i].suggestion + '</span>';
-      
-      messages += '<div class="flex-right">';
-      // If suggestion contains column list, add option to highlight columns in output table
-      if(suggestions[i].is_column_level_suggestion) {
-        messages += '<button ' +
-                      'class="btn primary suggestion-btn" ' + 
-                      'onclick=\'highlightColumns(this,' + JSON.stringify(suggestions[i].col_list) + ')\'>' +
-                         'Highlight in table' +
-                    '</button>';
-      }
-      // If suggestion contains row list, add option to highlight rows in output table
-      if(suggestions[i].is_row_level_suggestion) {
-        messages += '<button ' +
-                      'class="btn primary suggestion-btn" ' + 
-                      'onclick=\'highlightRows(this,' + JSON.stringify(suggestions[i].row_list) + ')\'>' +
-                         'Highlight in table' +
-                    '</button>';
-      }
-      // If suggestion contains json for alternate query, add option to run it
-      if('json' in suggestions[i]) {
-        console.log('suggestion json',JSON.stringify(suggestions[i].json));
-        messages += '<button ' +
-                      'class="btn primary suggestion-btn" ' + 
-                      'onclick=\'submitDebiasingQuery(' + JSON.stringify(suggestions[i].json) + ')\'>' +
-                         'Try this' +
-                    '</button>';
-      }
-      messages += '</div>';
-      
-      messages += '</div>';
-    }
-    document.querySelector('#suggestions-content').innerHTML = messages;
+    index = 0;
+    showSuggestion(index);
+    
+    hideNextButton();
```

Comment:
If there are more suggestions, shouldn't this be displayNextButton then?

Reply:
removing 
`
index = 0;
showSuggestion(index);
`
when the page loads initially, we should hide the buttons as the suggestion dropdown is closed. But when it opens it resets the index to zero.
So, these 2 lines are are redundant.